### ISSUE TITLE
Error when setting options.spritePath

### ISSUE THREAD

**Issue by hiulit** [drdk/grunt-dr-svg-sprites#48]
I'm working on my C: drive and I'm trying to create the sprites on the D: drive but I'm getting this error:

```bash
Fatal error: TypeError: 'undefined' is not a function (evaluating 'el.getBBox()')

  phantomjs://webpage.evaluate():5
  phantomjs://webpage.evaluate():35
  phantomjs://webpage.evaluate():35
Unable to calculate dimensions.
TypeError: 'null' is not an object (evaluating 'dimensions.width')
```
I've check for permission and it looks fine to me.

**Comment by rasmusfl0e** [comment]
That scenario hadn't crossed my mind.

Do any of the svg's get written to their intended destination?

**Comment by hiulit** [comment]
Yeah, the svg it's getting created but the png isn't. I guess it must have something to do with the creation of the png. And looking at the error thrown something to do with phantom.js

**Comment by hiulit** [comment]
I managed to move them using the `grunt copy` function, but it would be better if the plugin could do it by itself.
